coverage-report/pom.xml

```diff
@@ -41,7 +41,7 @@
             <dependency>
                 <groupId>com.google.api</groupId>
                 <artifactId>gax-httpjson</artifactId>
-                <version>0.114.1-SNAPSHOT</version> <!-- {x-version-update:gax-httpjson:current} -->
+                <version>2.29.1-SNAPSHOT</version> <!-- {x-version-update:gax-httpjson:current} -->
```

Comment:
Can you remove `{x-version-update:gax-httpjson:current}` altogether in favor of `version.gax`

Reply:
Ah, good idea! Updated in this file as well as the gax-bom. I noticed that we still have `gax-grpc`. Shall I remove this that reference as well (or maybe in a different PR)?